### ISSUE TITLE
Can't delete a Container version from history tab

### ISSUE THREAD

**Issue by joseorsini** [dotCMS/core#8624]
Reproduced in 3.3.
Reported by client. Support Ticket: https://my.dotcms.com/tickets/detail.dot?id=277efac7-aea6-4c70-9c38-709f6f65f051

Steps to reproduce:

1. Go to Containers portlet and open any Container, for example, "Default 2 (Page Content)" Container.
2. Add code in the Pre-loop or Post-loop sections. It can be a html comment tag.
3. Save and Publish.
4. Open again the same container and go to History Tab.
5. Delete the non-working version of it. 

Stacktrace
```
[17/02/16 08:29:40:136 EST] WARN util.JDBCExceptionReporter: SQL Error: 0, SQLState: 23503
[17/02/16 08:29:40:136 EST] ERROR util.JDBCExceptionReporter: ERROR: update or delete on table "inode" violates foreign key constraint "fk_cs_inode" on table "container_structures"
Detail: Key (inode)=(0b0f3aaf-956e-4797-9a3e-f6735df70755) is still referenced from table "container_structures".
[17/02/16 08:29:40:136 EST] ERROR impl.SessionImpl: Could not synchronize database state with session
[17/02/16 08:29:40:139 EST] WARN util.JDBCExceptionReporter: SQL Error: 0, SQLState: 25P02
[17/02/16 08:29:40:139 EST] ERROR util.JDBCExceptionReporter: ERROR: current transaction is aborted, commands ignored until end of transaction block
[17/02/16 08:29:40:139 EST] ERROR impl.SessionImpl: Could not synchronize database state with session
[17/02/16 08:29:40:144 EST] WARN util.JDBCExceptionReporter: SQL Error: 0, SQLState: 25P02
[17/02/16 08:29:40:144 EST] ERROR util.JDBCExceptionReporter: ERROR: current transaction is aborted, commands ignored until end of transaction block
[17/02/16 08:29:40:144 EST] ERROR impl.SessionImpl: Could not synchronize database state with session
```

Screenshot:
<img width="1227" alt="screen shot 2016-02-17 at 8 42 54 am" src="https://cloud.githubusercontent.com/assets/1717110/13111194/89632214-d552-11e5-9b16-bcb5e173dfb3.png">

**Comment by jcastro-dotcms** [comment]
PR: https://github.com/dotCMS/core/pull/8922

**Comment by bryanboza** [comment]
Fixed, tested on master-3.5 // Postgres // FF
